docs/introduction/understanding-docker.md

```diff
@@ -119,7 +119,7 @@ Docker images are the **build** component of Docker.
 Docker registries hold images. These are public or private stores from which you
 upload or download images. The public Docker registry is provided with the
 [Docker Hub](http://hub.docker.com). It serves a huge collection of existing
-images for your use. These can be images you create yourself or you can use
+images for your use. These can be images you created for yourself or 
```

Comment:
this changes the meaning of this sentence; if I create an image myself, it does not mean I only create it for my own use